$OpenBSD: patch-KdepimLibsConfig_cmake_in,v 1.2 2013/12/08 19:13:33 zhuk Exp $
kdepimlibs headers have references to Boost, upstream refuses badly to
add those to *_INCLUDE_DIRS.
--- KdepimLibsConfig.cmake.in.orig	Sat Feb  9 18:12:47 2013
+++ KdepimLibsConfig.cmake.in	Sat Feb  9 18:13:20 2013
@@ -15,7 +15,7 @@ set(KDEPIMLIBS_DATA_DIR            "@KDEPIMLIBS_DATA_D
 set(KDEPIMLIBS_DBUS_INTERFACES_DIR "@KDEPIMLIBS_DBUS_INTERFACES_DIR@")
 set(KDEPIMLIBS_DBUS_SERVICES_DIR   "@KDEPIMLIBS_DBUS_SERVICES_DIR@")
 set(KDEPIMLIBS_INCLUDE_DIR         "@KDEPIMLIBS_INCLUDE_DIR@")
-set(KDEPIMLIBS_INCLUDE_DIRS        "@KDEPIMLIBS_INCLUDE_DIR@" "@KDEPIMLIBS_INCLUDE_DIR@/KDE")
+set(KDEPIMLIBS_INCLUDE_DIRS        "@KDEPIMLIBS_INCLUDE_DIR@" "@KDEPIMLIBS_INCLUDE_DIR@/KDE" "@Boost_INCLUDE_DIR@")
 set(KDEPIMLIBS_LIB_DIR             "@KDEPIMLIBS_LIB_DIR@")
 set(KDEPIMLIBS_BIN_DIR             "@KDEPIMLIBS_BIN_DIR@")
 set(KDEPIMLIBS_LIBEXEC_DIR         "@KDEPIMLIBS_LIBEXEC_DIR@")
